src/components/AnonymousChat/child/ChannelPostNotificationStats.tsx

```diff
@@ -46,22 +47,22 @@ const ChannelPostNotificationStats = ({
   return (
     <View style={styles.descriptionContainer}>
       <View style={styles.iconContainer}>
-        <MemoIc_arrow_upvote_on style={styles.iconMargin} width={15} height={15} />
+        <IcUpvoteOn color={colors.darkBlue} width={15} height={15} />
         <Text style={styles.textVoteMargin}>{upvote}</Text>
       </View>
       <View style={styles.iconContainer}>
-        <MemoIc_arrow_down_vote_on style={styles.iconMargin} width={15} height={15} />
+        <IcDownvoteOn style={styles.iconMargin} width={15} height={15} />
         <Text style={styles.textVoteMargin}>{downvote}</Text>
       </View>
       <View style={styles.iconContainer}>
-        <MemoIc_comment style={styles.iconMargin} width={15} height={15} />
+        <IcComment style={styles.iconMargin} width={15} height={15} />
         <Text style={styles.textVoteMargin}>{comments}</Text>
       </View>
       <View style={styles.iconContainer}>
         {Number(block) > 0 ? (
           <Image source={Imageblock} style={styles.iconMargin} width={15} height={15} />
         ) : (
-          <MemoIc_block_inactive style={styles.iconMargin} width={15} height={15} />
+          <IcBlockInactive style={styles.iconMargin} width={15} height={15} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the new code, the `color` prop is passed to the `IcUpvoteOn` component but not to the other icon components (`IcDownvoteOn`, `IcComment`, `IcBlockInactive`). For consistency and to ensure that all icons are displayed with the correct color, consider passing the `color` prop to all icon components.

Also, the `style` prop is used inconsistently across different icon components. It's used with `IcDownvoteOn`, `IcComment`, and `IcBlockInactive` but not with `IcUpvoteOn`. To maintain consistency, consider applying the `style` prop to all icon components or none at all.

```diff
- <IcUpvoteOn color={colors.darkBlue} width={15} height={15} />
+ <IcUpvoteOn style={styles.iconMargin} color={colors.darkBlue} width={15} height={15} />

- <IcDownvoteOn style={styles.iconMargin} width={15} height={15} />
+ <IcDownvoteOn style={styles.iconMargin} color={colors.darkBlue} width={15} height={15} />

- <IcComment style={styles.iconMargin} width={15} height={15} />
+ <IcComment style={styles.iconMargin} color={colors.darkBlue} width={15} height={15} />

- <IcBlockInactive style={styles.iconMargin} width={15} height={15} />
+ <IcBlockInactive style={styles.iconMargin} color={colors.darkBlue} width={15} height={15} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->